src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,26 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## Aug 7, 2023
+
+### Roku SDK 1.0.0
+
+This is the first stable release of the Roku SDK, which exposes APIs to help developers to integrate the Adobe Experience Platform solutions with the Roku channel.
+
+Features:
+
+* **Adobe Experience Edge Network Integration:** Seamlessly send data to the Adobe Experience Edge Network. Optionally include an Experience Edge response callback using the API for greater control over data interactions.
+
+* **Automated ECID and Implementation Attachment:** Experience more efficient tracking and integration as ECID (Experience Cloud ID) and Implementation details are now automatically appended to every XDM Experience Event request.
+
+* **Custom Identity Synchronization:** Personalize user experiences effectively by syncing Custom Identities with each XDM Experience Event request. Unlock the potential of the Adobe Experience Edge Network while maintaining tailored interactions.
```

Comment:
Custom identity synchronization
For second mentions we can use simply .. of the Edge Network ...